easybuild/easyconfigs/p/PyTorch-bundle/PyTorch-bundle-1.12.1-foss-2022a-CUDA-11.7.0.eb

```
@@ -0,0 +1,59 @@
+easyblock = 'PythonBundle'
+
+name = 'PyTorch-bundle'
+version = '1.12.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://pytorch.org/'
+description = """PyTorch with compatible versions of official Torch extensions."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    ('RE2', '2022-06-01'),  # for torchtext
+]
+
+dependencies = [
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('PyTorch', version, versionsuffix),
+    ('Pillow-SIMD', '9.2.0'),  # for torchvision
+    ('SentencePiece', '0.1.97'),  # for torchtext
+    ('tqdm', '4.64.0'),  # for torchtext
+    ('double-conversion', '3.2.0'),  # for torchtext
+]
+
+use_pip = True
+
+exts_list = [
+    ('torchvision', '0.14.1', {
```

```suggestion
    ('torchvision', '0.13.1', {
```
if we are to follow compatibility matrix.